### PR TITLE
Ability to load a previous revision of a gist

### PR DESCRIPTION
**Pull request by Gaurav0**
Issue #66

### CODE REVIEW

File: tests/acceptance/git-revision-test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  const files = [
+    {
+      filename: "application.template.hbs",
+      content: "Hello, World!"
+    }
+  ];
+
+  runGist({
+    type: "revision",
```
Comment (joostdevries) [inline tests/acceptance/git-revision-test.js]: To me it looks weird to have a function called `runGist` which you can then pass an optional flag to run something else. Why not create `runGistRevision`?

File: app/serializers/gist-revision.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import GistSerializer from "./gist";
+
+export default GistSerializer.extend({
+  normalizeQueryResponse: function(store, primaryModelClass, payload, id, requestType) {
```
Comment (joostdevries) [inline app/serializers/gist-revision.js]: Out of curiousity, why is this needed?

Comment (Gaurav0) [inline app/serializers/gist-revision.js]: I'm querying for a record using two parameters. Ember Data expects an array back when you query. So I wrap the payload in an array.

File: tests/acceptance/git-revision-test.js (hunk 1 above)
Comment (Gaurav0) [inline tests/acceptance/git-revision-test.js]: You're right, it is weird. I did it because they were the mostly the same. I'll adjust. Thanks!
